src/Polytopes/Polyhedron/properties.jl

```diff
@@ -57,20 +70,17 @@ end
 Base.IteratorSize(::Type{PolyhedronFacePolyhedronIterator}) = Base.SizeUnknown()
 
 """
-    faces(as, P, face_dim)
+    faces(as::Type{T} = Polyhedron, P::Polyhedron, face_dim::Int)
 
-Return the faces of `P` of dimension `face_dim` as an iterator over the type of object given
-by `as`.
-Optional arguments for `as` include
-* `Polyhedron`/`Polyhedra`: Return the representation of a vertex as a point.
+Return the faces of `P` of dimension `face_dim` as an iterator over the type of
+object given by `as`.
 
-# Arguments
-- `as::Type{T}`: Element type of the returned iterator.
-- `P::Polyhedron`: A polyhedron.
-- `face_dim::Int`: Dimension of the desired faces.
+Optional arguments for `as` include
+* `Polyhedron`/`Polyhedra`.
 
-# Example
-An `Array` containing the six sides of the 3-dimensional cube can be obtained via the following input:
+# Examples
+An `Array` containing the six sides of the 3-dimensional cube can be obtained
```

Comment:
We have switched to referencing the type aliases `Vector` and `Matrix` instead of directly referencing `Array` just now, so...
```suggestion
A `Vector` containing the six sides of the 3-dimensional cube can be obtained
```